charts/blutgang/templates/poddisruptionbudget.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.podDisruptionBudget }}
+apiVersion: policy/v1beta1
```

this is not beta1 anymore.

https://kubernetes.io/docs/tasks/run-application/configure-pdb/

```suggestion
apiVersion: policy/v1
```